sdks/node-ts/src/apache_beam/coders/standard_coders.ts

```diff
@@ -80,4 +81,62 @@ export class IterableCoder<T> extends FakeCoder<Iterable<T>> {
         super();
         this.elementCoder = elementCoder;
     }
-}
\ No newline at end of file
+}
+CODER_REGISTRY.register(IterableCoder.URN, IterableCoder);
+
+export class StrUtf8Coder extends FakeCoder<String> {
+    static URN: string = "beam:coder:string_utf8:v1";
+    type: string = 'stringutf8coder';
+    encoder = new TextEncoder();
+    decoder = new TextDecoder();
+
+    constructor() {
+        super();
+    }
+
+    encode(element: String, writer: Writer, context: Context) {
+        writer.bytes(this.encoder.encode(element as string));
+    }
+
+    decode(reader: Reader, context: Context): String {
+        return this.decoder.decode(reader.bytes());
+    }
+}
+CODER_REGISTRY.register(StrUtf8Coder.URN, StrUtf8Coder);
+
+export class VarIntCoder extends FakeCoder<Long | Number | BigInt> {
+    static URN: string = "beam:coder:varint:v1";
+    encode(element: Number | Long | BigInt, writer: Writer, context: Context) {
+        writer.uint64(element as number);
+    }
+
+    decode(reader: Reader, context: Context): Long | Number | BigInt {
+        // TODO(pabloem): How do we deal with large integers?
+        return reader.uint64().low;
+    }
+}
+CODER_REGISTRY.register(VarIntCoder.URN, VarIntCoder);
+
+export class DoubleCoder extends FakeCoder<Number> {
```

Comment:
well if you actually implement Coder you don't need to extend FakeCoder :-) it is just a way to have the crashing methods filled in and we can delete it ASAP